docs/standard/datetime/datetime-support-in-system-text-json.md

```diff
@@ -0,0 +1,267 @@
+---
+title: DateTime and DateTimeOffset support in System.Text.Json
+description: An overview of how DateTime and DateTimeOffset types are supported in the System.Text.Json library.
+ms.technology: dotnet-standard
+author: layomia
+ms.author: laakinri
+ms.date: 07/22/2019
+helpviewer_keywords:
+  - "JSON DateTime"
+  - "DateTimeOffset"
+  - "Serializer, Utf8"
+  - "JSON Reader, JSON Writer, times"
+---
+# DateTime and DateTimeOffset support in System.Text.Json
+
+The <xref:System.Text.Json.JsonSerializer>, <xref:System.Text.Json.Utf8JsonReader>, <xref:System.Text.Json.Utf8JsonWriter>, and <xref:System.Text.Json.JsonElement> parse and write <xref:System.DateTime> and <xref:System.DateTimeOffset> text representations according to the extended profile of the ISO 8601-1:2019 format, e.g. 2019-07-26T16:59:57-05:00.
+
+Date and time data can be serialized with <xref:System.Text.Json.JsonSerializer>:
```

Comment:
Yeah "Date and time" is misguiding in some areas because it doesn't include time span. I've replaced with <xref:System.DateTime> and <xref:System.DateTimeOffset> where it makes sense.